examples/getting_started/custom_phase_transitions.py

```diff
@@ -47,7 +47,7 @@ def custom_phase_transition(state_pre: MX, state_post: MX, idx_1: int, idx_2: in
     The constraint such that: c(x) = 0
     """
 
-    return state_pre.cx_end[idx_1:idx_2, :] - state_post.cx[idx_1:idx_2, :]
+    return state_pre[idx_1:idx_2, :] - state_post[idx_1:idx_2, :]
```

Comment:
That is dangerous... If the user wants to call a biorbd function, they won't be able since CX can be an SX which won't work with biorbd. That said, the input hint (MX) suggest that it is the right type... Is this `mx_end` and `mx` which are sent or actually `cx_end` and `cx` as suggested by the old code?
It is working. 

That said, it does not allow anymore to use the code as everywhere else (for instance: `state_pre["q"].cx_end`, but instead relies on the fact that the user knows the indices, which we can assume. 

Is there a reason why you changed the input type (obviously the hint was wrong)?